### PR TITLE
Bug/plan a trip button

### PR DESCRIPTION
**Pull request by austinbrown-opspark**
Trips and Plan A Trip buttons only appear on landing page now, fixed home page button as well

### CODE REVIEW

File: client/components/Photos.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-  // const [photos, setPhotos] = useState([]);
+  const [photos, setPhotos] = useState([]);
 
   useEffect(() => {
     axios.get(`/photos/${currentTrip.id}`)
-      .then(() => {
-        // setPhotos(data);
+      .then(({ data }) => {
+        setPhotos(data);
+        console.log('photos from server', data);
```
Comment (jstrzesz) [inline client/components/Photos.js]: console.info

File: server/queries.js [hunk 2]
```diff
@@ -64,7 +64,8 @@ const addSplit = async (req, res) => {
     ower_id: user.user_id, recipient_id: purchaserId, amount, trip_id: tripId, item_id: item.id,
   }));
   await Promise.all(userObjs.map((user) => SplitOwedPayment.create(user)));
-  users = users.map((user) => User.findByPk(user.user_id, { raw: true }));
+  // users = users.map((user) => User.findByPk(user.user_id, { raw: true }));
```
Comment (jstrzesz) [inline server/queries.js]: remove

File: client/components/Photos.js (hunk 1 above)
Comment (austinbrown-opspark) [reply client/components/Photos.js]: This change has already been made in my most recent push

File: server/queries.js (hunk 2 above)
Comment (austinbrown-opspark) [reply server/queries.js]: This change has already been made in my most recent push

**Review by jldaggs88 — APPROVED** [review]

**Review by joshjnunez — APPROVED** [review]

**Pull request merged**